package.json

```diff
@@ -47,12 +47,13 @@
 		"node-fetch"
 	],
 	"devDependencies": {
+		"@ianwalter/dist": "^1.4.0",
+		"@ianwalter/puppeteer-helper": "^1.0.0",
```

Comment:
Yes, I added `dist` before I pulled in the changes that added Rollup and the UMD build. I will just use that instead.

`puppeteer-helper` is based off of a recipe in the AVA docs so I think if they wanted that to be an official helper they would have done it already. I don't think extensibility is an issue since you can pass any `puppeteer` config through `puppeteer-helper` as the second argument. The only thing that I could see becoming an issue is that `puppeteer-helper` controls the `puppeteer` version but I can't think of any good alternatives to that.

Reply:
In my opinion, `puppeteer-helper` should import `puppeteer` as a `peerDependency`, so its users can use their own Puppeteer version.

Reply:
IMO there's no need for that module. [10 additional lines of code](https://github.com/avajs/ava/blob/master/docs/recipes/puppeteer.md) is not much.

Reply:
Fair enough, I find it useful, but I'll replace it with a in-repo helper.